README.md

```diff
@@ -14,51 +14,51 @@ For deploying to a test server, see the README under **ansible** folder.
 
 ##Client Server Setup
 The application consists of a node/express software running the server and plain javascript/html on the client. The server
-powers reset endpoints, websocket communication, database interactions and the client website. 
+powers rest endpoints, websocket communication, database interactions and the client website. 
```

Comment:
I've never seen it with periods, as each letter isn't an abbreviation of an individual word: 

https://en.wikipedia.org/wiki/Representational_state_transfer

Would all-caps work (REST)?